src/modalities/__main__.py

```diff
@@ -283,11 +283,11 @@ def run(self, components: TrainingComponentsInstantiationModel):
         print_rank_0(f"Initialize Model at {datetime.now()}.")
         # save the config file to the checkpointing path
         if components.settings.cuda_env.global_rank == 0:
-            experiment_path = components.settings.paths.checkpointing_path / components.settings.experiment_id
+            experiment_path = components.settings.paths.checkpoint_saving_path / components.settings.experiment_id
             os.makedirs(experiment_path, exist_ok=True)
             shutil.copy(self.config_path, experiment_path / self.config_path.name)
 
-        evaluation_result_publisher, batch_processed_publisher = self.get_logging_publishers(
+        evaluation_result_publisher, process_publisher = self.get_logging_publishers(
```

Comment:
```suggestion
        evaluation_result_publisher, progress_publisher = self.get_logging_publishers(
```
Shouldn't this be called `progress_publisher`? Note that the variable gets passed to the `Trainer` and `Evaluator` as `progress_publisher` further below, too. So it would need to be changed there as well.

Reply:
done in 17b7a0cc3892371da0c1662a5479b844b5accc43